src/types/units.ts

```diff
@@ -0,0 +1 @@
+export type Units = 'standard' | 'metric' | 'imperial';
```

Comment:
множественное число используем для коллекци - enum / array / иногда объект. Units мог бы быть массивом юнитов. 
```suggestion
export type Unit = 'standard' | 'metric' | 'imperial';
```